app/models/user.rb

```diff
@@ -16,5 +16,13 @@ def skip_confirmation_method
     self.skip_confirmation!
   end
 
+  def destroy
+    update_attributes(suspended: true) unless suspended
```

Comment:
What're you trying to do here?

Reply:
Instead of the normal destroy method which would delete the user's data, we were tweaking it so that if called upon then suspended would be set to true. We were testing if we could call destroy from the suspend modal button.